### PR TITLE
Draft PR For [FRONTEND] Form page (desktop) task

### PR DESCRIPTION
**Pull request by DanielEmmanuel1**
Form Page Rework (Desktop Version)

Changes made:
- Redesigned form page layout according to Figma designs for desktop view
- Enhanced UI components:
  - Implemented new multiplier selector with dynamic gradient slider
  - Updated token selector styling
  - Improved balance cards display
  - Refined form input fields and styling
- Maintained backend integration and functionality
- Preserved all existing form submission logic

### CODE REVIEW

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Screenshot 2024-11-23 at 11 43 40
You didn't read guideline, I asked to create new branch and not using `main`

**Comment by DanielEmmanuel1** [comment]
I can create the branch now @djeck1432 
But apologies, creating a branch was never stated in the readme guideline

File: frontend/src/components/MultiplierSelector.jsx [hunk 1]
```diff
@@ -15,47 +15,66 @@ const MultiplierSelector = ({ setSelectedMultiplier, selectedToken, sliderValue
...
-    return (((maxMultiplier - actualValue + 1) - 1) / (maxMultiplier - 1)) * 100;
+    return ((maxMultiplier - actualValue + 1 - 1) / (maxMultiplier - 1)) * 100;
   }, [actualValue, maxMultiplier]);
 
-  if (isLoading) return <div className='slider-skeleton'>Loading multiplier data...</div>;
-  if (error) return <div className='error-message'>Error loading multiplier data: {error.message}</div>;
+  console.log('slider percentage', getSliderPercentage());
+  console.log('maxMultiplier', maxMultiplier);
+  console.log('actual value', actualValue);
+  console.log('slider value', sliderValue);
```
Comment (whateverfw) [inline frontend/src/components/MultiplierSelector.jsx:35]: remove logs

File: frontend/src/components/MultiplierSelector.jsx [hunk 2]
```diff
@@ -15,47 +15,66 @@ const MultiplierSelector = ({ setSelectedMultiplier, selectedToken, sliderValue
...
+
+  if (isLoading) return <div className="slider-skeleton">Loading multiplier data...</div>;
+  if (error) return <div className="error-message">Error loading multiplier data: {error.message}</div>;
+
+  const style = {
+    '--thumb-image': `url(${thumbImage})`,
+  };
```
Comment (whateverfw) [inline frontend/src/components/MultiplierSelector.jsx:43]: this should be in css, not as inline style

File: frontend/src/components/MultiplierSelector.jsx [hunk 3]
```diff
@@ -15,47 +15,66 @@ const MultiplierSelector = ({ setSelectedMultiplier, selectedToken, sliderValue
...
+                type="range"
+                min="1"
+                max={maxMultiplier}
+                step="0.1"
+                value={maxMultiplier - actualValue + 1}
+                onChange={handleMultiplierChange}
+                style={style}
+                className="multiplier-slider"
+              />
+            </div>
```
Comment (whateverfw) [inline frontend/src/components/MultiplierSelector.jsx:64]: slider' styles is not reflect figma design, please fix

File: frontend/src/components/TokenSelector.jsx
Comment (whateverfw) [inline frontend/src/components/TokenSelector.jsx:1]: please verify that token card are the same as in figma, currently styles differ

File: frontend/src/components/multiplier.css
Comment (whateverfw) [inline frontend/src/components/multiplier.css:1]: use css variables for colors everywhere

File: frontend/src/pages/forms/form.css
Comment (whateverfw) [inline frontend/src/pages/forms/form.css:1]: use css variables for colors

**Review by whateverfw — CHANGES_REQUESTED** [review]

File: frontend/src/components/BalanceCards.jsx
Comment (whateverfw) [inline frontend/src/components/BalanceCards.jsx:1]: balance cards styles is not correspond to figma

File: frontend/src/pages/forms/form.css
Comment (whateverfw) [inline frontend/src/pages/forms/form.css:1]: page background doesn't correspond to figma

**Review by whateverfw — CHANGES_REQUESTED** [review]

**Comment by djeck1432** [comment]
Duplicated PR

**Pull request closed without merging**